lib/absinthe/phase/document/complexity.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Absinthe.Phase.Document.Complexity do
+  @moduledoc false
+
+  # Analyses document complexity.
+
+  alias Absinthe.{Blueprint, Phase}
+
+  use Absinthe.Phase
+
+  @default_complexity 1
```

Comment:
Related to this, what should 0 represent (as a value returned by a complexity function)? Is there something "less" complex than `Map.get`? Would that be something that just returns a literal / constant value?